libraries/botbuilder-schema/botbuilder/schema/teams/_models_py3.py

```diff
@@ -707,12 +731,16 @@ class MessagingExtensionActionResponse(Model):
             "key": "composeExtension",
             "type": "MessagingExtensionResult",
         },
+        "cache_info": {"key": "cacheInfo", "type": "CacheInfo"},
     }
 
-    def __init__(self, *, task=None, compose_extension=None, **kwargs) -> None:
+    def __init__(
+        self, *, task=None, compose_extension=None, cache_info: CacheInfo, **kwargs
```

Comment:
shouldn't the new parameter be optional? And if sure that not (would be a breaking change) it should be placed before optional parameters

Reply:
definitely should be optional